runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -88,10 +89,7 @@ local function tags_lsp_to_vim(diagnostic, client_id)
       tags = tags or {}
       tags.deprecated = true
     else
-      vim.notify_once(
-        string.format('Unknown DiagnosticTag %d from LSP client %d', tag, client_id),
-        vim.log.levels.WARN
-      )
+      log.info('Unknown DiagnosticTag %d from LSP client %d', tag, client_id)
```

Comment:
```suggestion
      log.info(string.format('Unknown DiagnosticTag %d from LSP client %d', tag, client_id))
```

Otherwise it logs something like: `"Unknown diagnosticTag %d from LSP client %d"	1	20`